### PR TITLE
[BULK UPDATE] DocuTune - Rebranding links

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

**DO NOT MERGE THIS PULL REQUEST WHILE IN DRAFT STATUS!** Some of the changes to links and other content are not valid until after go-live.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: biztalk/adapters-and-accelerators/accelerator-swift/server-runtime-security.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ manager: "anneta"
...
\ No newline at end of file
+  Message Repair and New Submission imposes strict security policies by having checkpoints at each entry point into the message creation, repair, approval, and submission infrastructure. These checkpoints are tightly coupled with core functionality of Message Repair and New Submission and cannot be circumvented. While Message Repair and New Submission is designed to integrate seamlessly with the [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] form-signing functionality, it is also designed to be secure and robust enough to enforce authenticity and protection of SWIFT messages even if [!INCLUDE[btsInpathNoVersion](../../includes/btsinpathnoversion-md.md)] is not used and messages are submitted directly by end users to Message Repair and New Submission receive endpoints (SharePoint Web folders).
```
Comment (annmariehitchcock) [inline biztalk/adapters-and-accelerators/accelerator-swift/server-runtime-security.md:53]: Is this supposed to be a link to InfoPath?

**Comment by annmariehitchcock** [comment]
#docutune-wait

File: biztalk/adapters-and-accelerators/accelerator-swift/server-runtime-security.md (hunk 1 above)
Comment (alexbuckgit) [reply biztalk/adapters-and-accelerators/accelerator-swift/server-runtime-security.md:53]: No, this is just adding a new line to the end of the file.

**Comment by alexbuckgit** [comment]
#docutune-ready

**Review by MandiOhlinger — APPROVED** [review]

**Pull request merged**